### DISCUSSION TITLE
How do I compile tailwindcss to normal css

### DISCUSSION THREAD

**Discussion by H1Mohamed** [saadeghi/daisyui#2393] [Q&A] [answered]
I really loved the style of the input and toggle components But I dont want tailwindcss, I just want the styles for the elements, how can I do that?

**Comment by saadeghi** [comment, marked as answer]
You can use daisyUI without Tailwind CSS. Just the components.  
Here's an example which uses Post CSS plugins to add browser prefixes and also remove unused class names (Without Tailwind):  
https://stackblitz.com/edit/daisyui-postcss-no-tailwind?file=postcss.config.js

However, Without Tailwind, you should use custom CSS for layout, flexbox, spacing, etc because daisyUI only provides component class names.
